docs/source/usersguide/input.rst

```diff
@@ -374,37 +374,98 @@ OpenMOC does not place a limit on the hierarchical depth - or number of nested u
+   |   :width: 50 %                                         |   :width: 50 %                                         |
+   |   :align: right                                        |   :align: left                                         |
+   +--------------------------------------------------------+--------------------------------------------------------+
+
+The user may wish to caputre gradients in the moderator by adding rings in the moderator. The following code snipet repeats the scenario above but with 2 rings in the moderator.
+
+.. code-block:: python
+
+    # Subdivide the fuel region into 3 rings and 12 angular sectors
+    fuel.setNumRings(3)
+    fuel.setNumSectors(12)
+
+    # Subdivide the moderator region into 2 rings 16 angular sectors
+    moderator.setNumRings(2)
     moderator.setNumSectors(16)
-   
-The pin cell materials are illustrated on the left below, while the flat source regions with 3 equal volume rings and 8 sectors in the fuel, and 16 sectors in the moderator, are displayed on the right.
 
-.. _figure_fluxes:
+Again, the pin cell materials are illustrated on the left below, while the flat source regions are displayed on the right with 2 rings now present in the moderator.
+
+.. _figure_pin_cell_fsrs_moderator_rings:
+
+.. table::
+
+   +--------------------------------------------------------+--------------------------------------------------------+
+   | .. _figa:                                              | .. _figb:                                              |
+   |                                                        |                                                        |
+   | .. image:: ../../img/pin-cell-fsrs-1.png               | .. image:: ../../img/pin-cell-fsrs-moderator-rings.png |
+   |   :width: 50 %                                         |   :width: 50 %                                         |
+   |   :align: right                                        |   :align: left                                         |
+   +--------------------------------------------------------+--------------------------------------------------------+
+
+Lastly, the rings and sectors can be used to discretize regions between 2 ``ZCylinder`` objects, such as annular fuel. The following code snipet discretizes annular fuel into 3 rings and 12 sectors with the inner and outer moderators both discretized into 8 sectors with no rings.
+
+.. code-block:: python
+
+    # Subdivide the inner moderator region into 8 angular sectors
+    fuel.setNumSectors(8)
+
+    # Subdivide the annular fuel region into 3 rings and 12 sectors
+    fuel.setNumRings(3)
+    fuel.setNumSectors(12)
+
+    # Subdivide the outer moderator region into 8 angular sectors
+    moderator.setNumSectors(8)
+
+
+The annular pin cell materials are illustrated on the left below, with the resulting fuel and moderator discretization presented on the right.
```

Comment:
perhaps change `...illustrated on the left below` to `...illustrated below on the left`